cmd/snap/cmd_find.go

```diff
@@ -35,6 +35,21 @@ var longFindHelp = i18n.G(`
 The find command queries the store for available packages.
 `)
 
+func getPrice(prices map[string]float64, currency string) string {
+	// If there are no prices, then the snap is free
+	if len(prices) == 0 {
+		return "-"
+	}
+
+	// Look up the price by currency code
+	if val, ok := prices[currency]; ok {
+		return fmt.Sprintf("%.2f", val)
+	}
+
+	// Price was unavailable
```

Comment:
Prices are definitely available, because len(prices) > 0, and credit cards can handle pretty much any currency these days. So, I suggest something along those lines:
```
val, ok := prices[currency]
if !ok {
        val, ok = prices["USD"]
}
if !ok {
        currency = "ZZZ"
        for c, v := range prices {
                if c < currency {
                        currency, val = c, v
                }
        }
}

return fmt.Sprintf("%.2f%s", val, currency)